### PR TITLE
Bookstack deploy

### PR DESCRIPTION
**Pull request by BazRoe**
Deployment of the first version of the knowledgebase application

### CODE REVIEW

File: applications_dev/knowledgebase/mysql-knowledgebase.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  labels:
+    purpose: knowledgebase
+spec:
+  destination:
+    namespace: knowledgebase
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    path: ionos_dev/knowledgebase/mysql
+    repoURL: https://github.com/BazRoe/dome-gitops.git
```
Comment (ivanligot) [inline applications_dev/knowledgebase/mysql-knowledgebase.yaml:15]: @BazRoe wrong repoURL and targetRevision. should be https://github.com/DOME-Marketplace/dome-gitops.git and HEAD

**Comment by wistefan** [comment]
Hi, as a general question:
I see that the application is already deployed. How could that be the case? Its clearly against the idea of GitOps. If you(as the responsible operators) do not want to use a GitOps approach, I suggest to change that approach as a whole and document what ever approach you are more comfortable with.

**Comment by ivanligot** [comment]
> Hi, as a general question: I see that the application is already deployed. How could that be the case? Its clearly against the idea of GitOps. If you(as the responsible operators) do not want to use a GitOps approach, I suggest to change that approach as a whole and document what ever approach you are more comfortable with.

File: applications_dev/knowledgebase/mysql-knowledgebase.yaml (hunk 1 above)
Comment (BazRoe) [reply applications_dev/knowledgebase/mysql-knowledgebase.yaml]: Thank you, i've updated it

**Review by ivanligot — APPROVED** [review]

**Pull request merged**